src/adaptors/http1/http1_client.c

```diff
@@ -722,6 +724,17 @@ static int _client_rx_headers_done_cb(h1_codec_request_state_t *hrs, bool has_bo
     qd_compose_insert_string(props, hconn->cfg.address); // to
     qd_compose_insert_string(props, h1_codec_request_state_method(hrs));  // subject
     qd_compose_insert_string(props, hconn->client.reply_to_addr);   // reply-to
+    qd_compose_insert_null(props);                      // correlation-id
+    qd_compose_insert_null(props);                      // content-type
+    qd_compose_insert_null(props);                      // content-encoding
+    qd_compose_insert_null(props);                      // absolute-expiry-time
+    qd_compose_insert_null(props);                      // creation-time
+    qd_compose_insert_string(props, hconn->cfg.site);   // group-id
+
+    assert(hconn->client.reply_to_addr && strlen(hconn->client.reply_to_addr));  //remove me
```

Comment:
Is this intended for inclusion with this patch?

Reply:
Not the assert! I must have accidentally included that when merging conflicts in the rebase. Will remove.